scripts/answerbtn.js

```diff
@@ -1,19 +1,19 @@
 //answer button
 
-const answerBtn = document.querySelector('[data-js="answerBtn"]');
+const answerBtn = document.querySelectorAll('[data-js="answerBtn"]');
 
-const answer = document.querySelector('[data-js="answer"]');
+const answer = document.querySelectorAll('[data-js="answer"]');
 
-answerBtn.addEventListener("click", () => {
-  console.log("Show Answer");
-  answer.classList.toggle("quiz-card__answer--hidden");
-});
+const showAnswer = (num) => {
+  for (let i = 0; i < answer.length; i++) {
+    if (i === num) {
+      answer[num].classList.toggle("quiz-card__answer--hidden");
+    }
+  }
+};
```

Comment:
Ist eine for-Schleife hier sinnvoll? Ich hab noch nicht verstanden, was die Länge der Antwort mit dem Anzeigen der Antwort zu tun hat... ^^